io/plugins/eu.esdihumboldt.hale.io.jdbc.msaccess/src/eu/esdihumboldt/hale/io/jdbc/msaccess/MsAccessDataReader.java

```diff
@@ -0,0 +1,50 @@
+package eu.esdihumboldt.hale.io.jdbc.msaccess;
+
+import java.io.File;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class MsAccessDataReader {
+
+	public static String getFirstData() {
```

Comment:
This is intended for testing, right?
So I would move it to the test bundle. Also, you will need some additional logic to make it work when your test database is located inside the JAR.